language/bytecode-verifier/bytecode-verifier-tests/src/unit_tests/bounds_tests.rs

```diff
@@ -124,6 +129,20 @@ fn invalid_locals_id_in_call() {
     m.freeze().unwrap_err();
 }
 
+#[test]
+fn script_invalid_locals_id_in_call() {
+    use Bytecode::*;
+
+    let mut s = basic_test_script();
+    s.function_instantiations.push(FunctionInstantiation {
+        handle: FunctionHandleIndex::new(0),
+        type_parameters: SignatureIndex::new(1),
+    });
+    let func_inst_idx = FunctionInstantiationIndex(s.function_instantiations.len() as u16 - 1);
+    s.code.code = vec![CallGeneric(func_inst_idx)];
+    s.freeze().unwrap_err();
```

Comment:
It might be useful to check that we get the expected error code (here and other tests). It looks like the existing tests don't do this, but shouldn't hurt to be more specific (especially given that the error codes need to be stable).

Reply:
Good idea, thanks! See #8537.